Configure.cmake

```diff
@@ -32,25 +25,34 @@ if (ENFORCE_TESTER3 AND NOT SLEEF_OPENSSL_FOUND)
   message(FATAL_ERROR "ENFORCE_TESTER3 is specified and OpenSSL not found")
 endif()
 
-if (LIB_MPFR)
-  find_path(MPFR_INCLUDE_DIR
-    NAMES mpfr.h
-    ONLY_CMAKE_FIND_ROOT_PATH)
-endif(LIB_MPFR)
-
-if (LIBFFTW3)
-  find_path(FFTW3_INCLUDE_DIR
-    NAMES fftw3.h
-    ONLY_CMAKE_FIND_ROOT_PATH)
-endif(LIBFFTW3)
-
-if (NOT LIBM)
-  set(LIBM "")
-endif()
+if (NOT SLEEF_CLANG_ON_WINDOWS)
```

Comment:
I don't get this - why do you need to customize this for clang on windows? If the libraryes are not available, the variables below (`LIB_MPFR`, `LIBFFTW3`, ..) would evaluate to false and teh corresponding compilation flags will not be set. Is that breaking things?

Reply:
CMake finds libraries for Cygwin, and this causes build errors.
We need to check the builds with Cygwin testers.

Reply:
May I ask you to add this explanation as a comment? It will help people like me who doesn't know much about compiling on windows in understanding why there is this `if not clang_on_windows` condition.

Reply:
I added the comment.